src/freenet/config/SubConfig.java

```diff
@@ -172,47 +172,53 @@ public int getInt(String optionName) {
 		synchronized(this) {
 			o = (IntOption) map.get(optionName);
 		}
-		return o.getValue();
+		// return fallback value for ignored options (null). This avoids breaking plugins which try to get ignored options.
+		return o == null ? -1 : o.getValue();
```

Comment:
Do you know of any plugins that fail because of the addition of ignored options?

Reply:
My fred broke during development because I had a place left that accessed the removed option.

I don’t know about plugins that get these options, but this change enables us to set options as ignored without having to worry about plugins.